Dockerfile

```diff
@@ -7,6 +7,7 @@ ARG BUILDARGS
 # Build all the binaries at once, so that the final targets don't require having
 # Go installed to build each of them.
 WORKDIR /src
+# COPY . .
```

Comment:
?

do note that instead of doing the COPY, a bind mount is used in line 17
`--mount=type=bind,target=.`

Reply:
Do you know if this works with docker-compose too? or only with docker?

Reply:
both. docker-compose uses docker to build images